course-v2/assets/js/mobile_course_drawers.ts

```diff
@@ -0,0 +1,35 @@
+export const initCourseDrawersClosingViaSwiping = () => {
+  elementSwiping("mobile-course-nav", "mobile-course-nav-toggle", "L")
+  elementSwiping("course-info-drawer", "mobile-course-info-toggle", "R")
+}
+
+/**
+ * It adds touch event listener to the element and clicks the button when swiped in the mentioned direction.
+ *
+ * @param {string} elementId element on which touch eventlistenter is added.
+ * @param {string} buttonId This button will be clicked on swiping
+ * @param {string} swipeDirection L= Swipe left, R= Swipe Right
+ */
+const elementSwiping = (
+  elementId: string,
+  buttonId: string,
+  swipeDirection: string
+) => {
+  const element = document.getElementById(elementId)
+  let touchstartX = 0
+  let touchendX = 0
+
+  element?.addEventListener("touchstart", e => {
+    touchstartX = e.changedTouches[0].screenX
+  })
+
+  element?.addEventListener("touchend", e => {
+    touchendX = e.changedTouches[0].screenX
+    if (swipeDirection === "R") {
+      // checking if its a right swipe
+      if (touchendX > touchstartX) document.getElementById(buttonId)?.click()
+    } else {
+      if (touchendX < touchstartX) document.getElementById(buttonId)?.click()
+    }
```

Comment:
I might be missing something but why not use use "!==" rather than checking inequalities.

Reply:
inequalities are used to identify the swipe direction. 
if `touchendX > touchstartX` then its a right swipe.
if `touchendX < touchstartX` then its a left swipe.

We can't use  `!==`, as it will enable the drawer to be closed in both, left and right directions but actually, it should be closed only in one direction. (course menu drawer should only be closed in left direction whereas course info drawer should only be closed in right direction).

Try replacing the inequalities with "!==" and you will notice what I'm talking about